ehrapy/api/io/read.py

```diff
@@ -263,12 +332,8 @@ def _read_text(  # noqa:C901
         )
 
     @staticmethod
-    def _df_to_anndata(
-        df: pd.DataFrame, index_column: Union[str, Optional[int]], columns_obs_only: Optional[List[Union[str]]]
-    ) -> AnnData:
+    def _df_to_anndata(df: pd.DataFrame, columns_obs_only: Optional[List[Union[str]]]) -> AnnData:
```

Comment:
It might make sense to expose this function via the API for people who write custom data parsers.

Reply:
@Imipenem can we expose this?